core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala

```diff
@@ -138,8 +140,9 @@ object SparkSubmit {
     (clusterManager, deployMode) match {
       case (MESOS, CLUSTER) =>
         printErrorAndExit("Cluster deploy mode is currently not supported for Mesos clusters.")
-      case (_, CLUSTER) if args.isPython =>
-        printErrorAndExit("Cluster deploy mode is currently not supported for python applications.")
+      case (STANDALONE, CLUSTER) if args.isPython =>
+        printErrorAndExit("Standalone-Cluster deploy mode is currently not supported" +
```

Comment:
Nit: to mirror the above messages a little better, can this be "Cluster deploy mode is currently not supported for python applications on Standalone clusters"?